docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,105 @@
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+### Advisors
+
+
+- **Advisor checks enabled by default**
+
+    Starting with the previous release and continuing with this one, we have added significant improvements to the Advisors Checks functionality in performance, assessment coverage, and user experience.
+    
+    As a mature and generally useful feature, this option is now enabled by default for easier access to automatic checks and better insight into database health and performance, delivered by Percona Platform.
+    
+    !!! note alert alert-primary "Note"
+        Upgrading to PMM will automatically enable this feature for existing PMM instances. You can disable it at any time from your PMM dashboard on the **Advanced Settings** page.
+
+- **Run individual advisor checks**
+
+    In addition to running all available advisors at once, you now have the option to run each advisor check individually. 
+
+    This gives you more granular control over the checks on your connected databases. Running checks individually also means that you get the results for relevant advisors faster and that you can focus on resolving failed checks one at a time. For more information, see [Working with Advisor checks](https://docs.percona.com/percona-monitoring-and-management/how-to/advisors.html).
+
+
+- **Enhanced Advisor checks**
+
+    PMM 2.28 includes a new major version of Advisors that features some important enhancements. The most significant changes are:
+
+    * Support for multiple queries
+    * Support for Victoria Metrics as a data source
+
+    In a nutshell, these changes will allow experts to create more intelligent advisor checks so that we can continue delivering more value to your connected PMM instances.
+
+    The file format in which Advisors checks are written has been updated to support the new functionality provided by the Advisors service part of Percona Platform.
+
+    This is a  breaking change, so we recommend upgrading your PMM instance to benefit from these enhancements. For more information, see [Develop Advisors](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks.html). 
+
+### Ubuntu 22.04 LTS support
+
+
+### Components upgrade
+  - **VictoriaMetrics**: VictoriaMetrics has been upgraded to [1.76.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.76.1).
+  - **Node exporter**: Node Exporter has now been updated to `1.3.1`.
+
+    !!! caution alert alert-warning "Important"
+        If you customized the disabled collectors, the list could change. Check the available collectors in [Documentation](https://github.com/percona/node_exporter/blob/main/README.md#collectors).
```

Comment:
Can we reference  "disabled collectors" to the specific section in pmm-admin documentation?